namui-cli/src/util/debug_println.rs

```diff
@@ -8,5 +8,5 @@ macro_rules! debug_println {
 #[cfg(not(feature = "cli_debug"))]
 #[macro_export]
 macro_rules! debug_println {
-    ($( $args:expr ),*) => {};
+    ($( $args:expr ),*) => {$( let _ = $args; )*};
```

Comment:
This is very important. It make compiler check whether the args are good or not in not-cli_debug mode too.
Previously, this macro just ignored args so compiler didn't check about args.